example/rails-51/config/database.yml

```diff
@@ -5,6 +5,11 @@
 #   gem 'sqlite3'
 #
 default: &default
+  adapter: sqlite3
+  pool: 5
+  timeout: 5000
```

Comment:
(GitHub won't let me comment further below) The database entry for each section using the sqlite3 adapter should ideally be a file in the db directory with an extension of sqlite3 (as opposed to just "default" in the Rails root).

Reply:
We don't make use of sqlite database in our example as far as I can see, so don't think we need to update this.

Reply:
We don't make use of these sqlite databases in our examples as far as I can see - I don't see any benefit in updating this.